core/eolearn/tests/test_eodata_io.py

```diff
@@ -314,6 +321,7 @@ def assert_data_equal(data1: Any, data2: Any) -> None:
             ),
         ),
         (FeatureIOJson, {"test": "test1", "test3": {"test": "test1"}}),
+        (FeatureIOTimeStamp, [datetime.datetime(2017, 1, 1, 10, 4, 7), datetime.datetime(2017, 1, 4, 10, 14, 5)]),
```

Comment:
Do you think it would be worthwhile to add a test for an empty list? Perhaps one for an empty dictionary in FeatureIOJson as well...